### PR TITLE
Fixed an error in the description for handling Issues in CDAP.

### PR DESCRIPTION
**Pull request by awholegunch**
Deleted some lower-level READMEs that were either incorrect (goodbye gradle!) or of low-value.
Convert all upper-level README and similar files to reST from markdown.

Fixes https://issues.cask.co/browse/CDAP-991

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -34,71 +41,81 @@ Developers can build and test their applications end-to-end in a full-stack, sin
...
+-------------
 
 To install and use CDAP and its included examples, there are a few simple prerequisites:
 
-  1. JDK 6 or JDK 7 (required to run CDAP; note that $JAVA_HOME should be set)
-  2. Node.js (from v0.8.16; required to run the CDAP UI)
-  3. Apache Maven 3.0+ (required to build the example applications)
+1. JDK 6 or JDK 7 (required to run CDAP; note that $JAVA_HOME should be set)
+2. Node.js (from v0.8.16; required to run the CDAP UI)
+3. Apache Maven 3.0+ (required to build the example applications)
```
Comment (wolf31o2) [inline README.rst]: It's Maven 3.1, now... The new UI introduced a new build dependency which requires Maven 3.1

File: README.rst [hunk 2]
```diff
@@ -34,71 +41,81 @@ Developers can build and test their applications end-to-end in a full-stack, sin
...
 
-We have a simple pull-based development model with a consensus-building phase, similar to Apache's
-voting process. If you’d like to help make CDAP better by adding new features, enhancing existing
-features, or fixing bugs, here's how to do it:
+We have a simple pull-based development model with a consensus-building phase, similar to
+Apache's voting process. If you’d like to help make CDAP better by adding new features,
+enhancing existing features, or fixing bugs, here's how to do it:
 
-1. If you are planning a large change or contribution, discuss your plans on the `cask-cdap-dev`
+1. If you are planning a large change or contribution, discuss your plans on the ``cask-cdap-dev``
```
Comment (wolf31o2) [inline README.rst]: This is just `cdap-dev`... https://groups.google.com/forum/#!forum/cdap-dev

File: README.rst [hunk 3]
```diff
@@ -111,32 +128,35 @@ features, or fixing bugs, here's how to do it:
 
 Thanks for helping to improve CDAP!
 
-### Filing Issues
-JIRA for filing [Issues](http://issues.cask.co)
+Filing Issues
+-------------
+Website for filing `Issues <http://issues.cask.co>`__.
```
Comment (wolf31o2) [inline README.rst]: Better URL: `https://issues.cask.co/browse/CDAP`

File: README.rst (hunk 1 above)
Comment (awholegunch) [inline README.rst]: Is that dependency just for building CDAP? For building just the examples, would 3.0+ work? (This is not the requirements for building CDAP...)

File: README.rst (hunk 2 above)
Comment (awholegunch) [inline README.rst]: Fixed.

File: README.rst (hunk 1 above)
Comment (wolf31o2) [inline README.rst]: I am not aware of the examples needing Maven 3.1+ but CDAP itself does.

File: README.rst (hunk 3 above)
Comment (awholegunch) [inline README.rst]: Fixed.

**Comment by awholegunch** [comment]
This PR also updates the READMEs to require JDK 7.

**Comment by sreevatsanraman** [comment]
LGTM :+1:

**Pull request merged**